include/swift/AST/DiagnosticsParse.def

```diff
@@ -131,17 +131,17 @@ ERROR(lex_unicode_escape_braces,none,
 ERROR(lex_illegal_multiline_string_start,none,
       "multi-line string literal content must begin on a new line", ())
 ERROR(lex_illegal_multiline_string_end,none,
-      "multi-line string literal content must not be on the delimiter's line", ())
-ERROR(lex_insufficient_string_indent,none,
-      "string literal content is insufficiently indented", ())
-ERROR(lex_unexpected_tab_in_string_indent,none,
-      "string literal content is indented with a tab where a space is expected", ())
-ERROR(lex_unexpected_space_in_string_indent,none,
-      "string literal content is indented with a space where a tab is expected", ())
+      "multi-line string literal closing delimiter must begin on a new line", ())
+ERROR(lex_inconsistent_string_indent,none,
+      "line does not start with same indentation as closing delimiter of multi-line string literal", ())
```

Comment:
Hmm, I think the note idea is great, but I feel like there are so many notes now. Can I make a few suggestions?

* This error message doesn't need to be so long; the fix-it explains exactly what to do already--this can just be the original "string literal content is insufficiently indented".